lisp/ess-custom.el

```diff
@@ -719,6 +719,38 @@ some.function(arg1,
 
 ")
 
+(defvar ess-arg-function-offset-continued nil
+  "Extra indent for function arguments when ( is folowed by arguments and
+a new line.
+
+If nil, the statements are indented at open-parenthesis following foo:
```

Comment:
Would you please add a note here that this variable is here only for the compatibility with R studio and is not recommended by ESS developers. 

Also please enclose "foo" in quotes of some kind.